cookbook-react-native/recipes/pickers/dateTimePicker/constants.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { TouchableOpacity, Image, Text } from 'react-native';
+import i18next from 'i18next';
+
+import './i18n';
+import styles from './style';
+import { ITouchButton } from './interfaces';
+
+const toLocaleTimeString: string = 'toLocaleTimeString';
+const toISOString: string = 'toISOString';
+
+export const FULL_HOUR = (is24hs: boolean) => (is24hs ? 'en_GB' : '');
+
+export const hideDatePicker = (setDatePickerVisibility: Function) => {
```

Comment:
we generally not define the typing of a function as `Function`. I would suggest creating a type of an arrow function like
`type thisFunc = (isVisible: boolean) => void;`

Reply:
changed